yt_dlp/extractor/neteasemusic.py

```diff
@@ -203,91 +154,151 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
+            **lyric_data,
+            **traverse_obj(info, {
+                'title': ('name', {str}),
+                'timestamp': ('album', 'publishTime', {lambda i: int_or_none(i, scale=1000)}),
+                'thumbnail': ('album', 'picUrl', {url_or_none}),
+                'duration': ('duration', {lambda i: int_or_none(i, scale=1000)}),
+            }),
         }
 
 
 class NetEaseMusicAlbumIE(NetEaseMusicBaseIE):
     IE_NAME = 'netease:album'
     IE_DESC = '网易云音乐 - 专辑'
     _VALID_URL = r'https?://music\.163\.com/(#/)?album\?id=(?P<id>[0-9]+)'
-    _TEST = {
+    _TESTS = [{
+        'url': 'https://music.163.com/#/album?id=133153666',
+        'info_dict': {
+            'id': '133153666',
+            'title': '桃几的翻唱',
+            'upload_date': '20210913',
+            'description': '桃几2021年翻唱合集',
+            'thumbnail': r're:^http.*\.jpg',
+        },
+        'playlist_mincount': 13,
+    }, {
         'url': 'http://music.163.com/#/album?id=220780',
         'info_dict': {
             'id': '220780',
-            'title': 'B\'day',
+            'title': 'B\'Day',
+            'upload_date': '20060904',
+            'description': 'md5:71a74e1d8f392d88cf1bbe48879ad0b0',
+            'thumbnail': r're:^http.*\.jpg',
         },
         'playlist_count': 23,
-        'skip': 'Blocked outside Mainland China',
-    }
+    }]
 
     def _real_extract(self, url):
         album_id = self._match_id(url)
-
-        info = self.query_api(
-            'album/%s?id=%s' % (album_id, album_id),
-            album_id, 'Downloading album data')['album']
-
-        name = info['name']
-        desc = info.get('description')
+        webpage = self._download_webpage(f'https://music.163.com/album?id={album_id}', album_id)
+
+        songs = self._search_json(
+            r'<textarea id="song-list-pre-data" style="display:none;">', webpage, 'metainfo', album_id,
+            end_pattern=r'</textarea>', contains_pattern=r'\[(?s:.+)\]')
+        metainfo = {
+            'title': self._og_search_property('title', webpage, 'title', fatal=False),
+            'description': clean_html(self._search_regex(
+                r'(?:<div id="album-desc-dot".*)?<div id="album-desc-(?:dot|more)"[^>]*>(.*?)</div>', webpage,
+                'description', flags=re.S, fatal=False)),  # match album-desc-more or fallback to album-desc-dot
+            'thumbnail': self._og_search_property('image', webpage, 'thumbnail', fatal=False),
+            'upload_date': unified_strdate(self._html_search_meta('music:release_date', webpage, 'date', fatal=False)),
+        }
         entries = [
-            self.url_result('http://music.163.com/#/song?id=%s' % song['id'],
-                            'NetEaseMusic', song['id'])
-            for song in info['songs']
+            self.url_result(
+                f"http://music.163.com/#/song?id={song['id']}", NetEaseMusicIE, song['id'], song.get('name'))
+            for song in songs
         ]
```

Comment:
This is a commonly repeated pattern in these extractors... could it be a baseclass method? And if it's too difficult to generalize an entries generator function, we could use a common method for `return url_result(...)`

Otherwise, outer quotes should be single quotes and inner quotes should be double quotes
```suggestion
            self.url_result(
                f'http://music.163.com/#/song?id={song["id"]}', NetEaseMusicIE, song['id'], song.get('name'))
            for song in songs
        ]
```

Reply:
https://github.com/yt-dlp/yt-dlp/pull/8181/commits/b14de8dc3a03039ee7a29db448682612be44fb52